.github/workflows/e2e-windows.yml

```diff
@@ -137,3 +135,15 @@ jobs:
         shell: pwsh
         run: |
           .\constellation.exe iam destroy --debug -y
+
+      - name: Notify about failure
+        if: |
+          failure() && github.event_name == 'workflow_call'
```

Comment:
should look more like: https://github.com/edgelesssys/constellation/blob/main/.github/workflows/e2e-upgrade.yml#L298-L300

- check for main branch
- pass if event was scheduled to avoid notifications on dispatch triggered workflow caller

Reply:
The second point is still an issue.